static/app/views/performance/newTraceDetails/traceHeader/breadcrumbs.tsx

```diff
@@ -69,20 +71,38 @@ function getBreadCrumbTarget(
   };
 }
 
-function getPerformanceBreadCrumbs(organization: Organization, location: Location) {
+function getPerformanceBreadCrumbs(
+  organization: Organization,
+  location: Location,
+  view: DomainView
+) {
   const crumbs: Crumb[] = [];
 
+  const performanceUrl = getPerformanceBaseUrl(organization.slug, view, true);
```

Comment:
nit: hmm, getPerformanceBaseUrl already calls `normalizeUrl` and prefixes the view with `/organizations/${orgSlug}` by default (when the `bare` option is not set to true).

Further down, looks like we're calling `getBreadCrumbTarget` on `performanceUrl` which does something very similar. The only difference is that we're also carrying over some params from `location.query`.

I wonder if we can simplify by removing the `bare` logic from `getPerformanceBaseUrl`, since it's only used in one place (this file). And then instead of calling `getBreadCrumbTarget` on line 97, we just construct the breadcrumb in place using `performanceUrl`. ie: `to: {pathname: performanceUrl, query: {...omit(location.query, ['node', 'fov', 'timestamp', 'eventId'])}}`

Reply:
Yeah, that's way better! Truthfully I didn't consider this option when implementing, because there's a bunch of other areas in our app that also require the "bare" url, so i figured to just keep things as is where possible.

Does it sound good to you to keep the `bare` logic here, and in a subsequent PR I can try to remove it here and other areas of our app.